miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -283,6 +284,11 @@ public static SampleAdditionalInfo to(SampleAdditionalInfoDto from) {
       sampleClass.setSampleClassId(from.getSampleClassId());
       to.setSampleClass(sampleClass);
     }
+    if (from.getParentId() != null) {
```

Comment:
This doesn't seem right. Since we've got the parent id we can lookup the actual parent. (Not here in this function, but later when it's required.)